redpen-core/src/main/java/cc/redpen/RedPen.java

```diff
@@ -138,7 +139,6 @@ public Document parse(DocumentParser parser, String content) throws RedPenExcept
     public Map<Document, List<ValidationError>> validate(List<Document> documents) {
         Map<Document, List<ValidationError>> docErrorsMap = new HashMap<>();
         documents.forEach(e -> docErrorsMap.put(e, new ArrayList<>()));
-        initializeValidators();
```

Comment:
Ah.. I see!

I think then I will need to get the UI to parse a fake document at startup to load the relevant validator configurations. Let me see how that goes.